evalml/pipelines/components/estimators/classifiers/lightgbm_classifier.py

```diff
@@ -30,7 +30,7 @@ class LightGBMClassifier(Estimator):
     SEED_MIN = 0
     SEED_MAX = SEED_BOUNDS.max_bound
 
-    def __init__(self, boosting_type="gbdt", learning_rate=0.1, n_estimators=100, max_depth=0, num_leaves=31, min_child_samples=20, n_jobs=-1, random_state=0, bagging_fraction=0.9, bagging_freq=0, **kwargs):
+    def __init__(self, boosting_type="gbdt", learning_rate=0.1, n_estimators=100, max_depth=0, num_leaves=31, min_child_samples=20, n_jobs=-1, random_state=0, **kwargs):
```

Comment:
How come we're getting rid of the defaults for these parameters? I think the warnings would still be suppressed if they're included.

Reply:
@freddyaboulton yea you're right, i think. If I just override `subsample` and `subsample_freq` as None, it should work